go/worker/common/config.go

```diff
@@ -42,6 +42,9 @@ var (
 	// runtime IDs to corresponding resource paths (type of the resource depends on the
 	// provisioner).
 	CfgRuntimePaths = "worker.runtime.paths"
+	// CfgRuntimeSignatures configures signatures for supported runtimes.  The value should
+	// be a map of runtime IDs to correspondign resource paths.
+	CfgRuntimeSignatures = "worker.runtime.signatures"
```

Comment:
Since this is SGX-specific, maybe move this to `worker.runtime.sgx.signatures` as the above loader option?